ecosystem/sep-0001.md

```diff
@@ -151,7 +151,51 @@ Replace `$PUBLIC_KEY` with the account's public key, `$CODE` with your asset cod
 
 Note: `fixed_number`, `max_number`, and `is_unlimited` are mutually exclusive issuance policies. Include exactly one of those fields.
 
-Alternately, `stellar.toml` can link out to a separate TOML file for each currency by specifying `toml="https://DOMAIN/.well-known/CURRENCY.toml"` as the currency's only field.
+
+### External Currency Specification
+There are two (compatible) ways to reference currency entries in separate files:
```

Comment:
It's unclear to me what compatible means here, regardless it seems superfluous to me.

@Shaptic Was there some meaning you were intending to communicate?

```suggestion
There are two ways to reference currency entries in separate files:
```

Reply:
I'm trying to say that the two methods are compatible with each other, i.e. they're not mutually exclusive. Maybe `(mutually compatible)` is a better way to convey that?

Reply:
If this is important I would say it in a sentence rathe than bottling it into a word or two. i.e.:
```suggestion
`stellar.toml`s may store all or some of their `CURRENCIES` in separate files. There are two methods to reference `CURRENCIES` entries that are stored in separate files. Both methods can be used on their own, or together.
```

Reply:
Good idea, done!